tests/web/_bootstrap.php

```diff
@@ -1,4 +1,4 @@
 <?php
 // Here you can initialize variables that will for your tests
 
-\Codeception\Util\Autoload::registerSuffix('Steps', __DIR__.DIRECTORY_SEPARATOR.'_steps');
+//\Codeception\Util\Autoload::registerSuffix('Steps', __DIR__.DIRECTORY_SEPARATOR.'_steps');
```

Comment:
looks like this broke web tests

Reply:
That method was marked as deprecated.

Reply:
It would be better to rename `RootWatcherSteps` class to `\WebGuy\Steps\RootWatcher`, right?
In a similar manner as `FunctionalHelper` was renamed to `Helper\Functional` few versions ago.

Reply:
Ok, I will try to take this